hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-web-proxy/src/test/java/org/apache/hadoop/yarn/server/webproxy/TestWebAppProxyServlet.java

```diff
@@ -361,8 +362,8 @@ public void testAppReportForEmptyTrackingUrl() throws Exception {
         proxyConn.getURL().toString().equals(appAddressInRm));
 
     //set AHS_ENBALED = true to simulate getting the app report from AHS
-    configuration.setBoolean(YarnConfiguration.APPLICATION_HISTORY_ENABLED,
-        true);
+    configuration.setBoolean(YarnConfiguration.APPLICATION_HISTORY_ENABLED, true);
+    proxy.proxy.appReportFetcher.setAhsAppPageUrlBase(configuration);
```

Comment:
Not sure why checkstyle complains about this.

Reply:
Because the code style of original code is wrong, I will fix all.